src/contracts/core/DelegationManager.sol

```diff
@@ -875,10 +838,6 @@ contract DelegationManager is
                  */
                 eigenPodManager.removeShares(staker, sharesToDecrement);
             } else {
-                require(
-                    staker == withdrawer || !strategyManager.thirdPartyTransfersForbidden(strategies[i]),
```

Comment:
we still need to make sure the staker is the withdrawer

Reply:
moved above, instead of checking per strategy